### PR TITLE
[Ruby]-3.0 EOL- Images Issue #998

### PR DESCRIPTION
**Pull request by gauravsaini04**
**Dev container name**:
 
 * Ruby
 
 **Description**:
 
 This PR achieved the following :-
 * `Ruby` Devcontainer's EOL changes for 3.0 Version
  
 _Changelog_:
 * updated `Dockerfile`
 * updated `manifest.json`
 * updated `readme.md`
 
 **Checklist**:
 
 * [x]   Checked that applied changes work as expected

### CODE REVIEW

**Review by samruddhikhandale — APPROVED** [review]
Looks good, thanks!

**Review by MrNice78 — COMMENTED** [review]
[README.md](https://github.com/devcontainers/images/files/15172551/README.md)

File: src/ruby/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 | *Categories* | Core, Languages |
 | *Image type* | Dockerfile |
 | *Published images* | mcr.microsoft.com/devcontainers/ruby |
-| *Available image variants* | 3 / 3-bookworm, 3.3 / 3.3-bookworm, 3.2 / 3.2-bookworm, 3.1 / 3.1-bookworm, 3.0 / 3.0-bullseye, 3-bullseye, 3.3-bullseye, 3.2-bullseye, 3.1-bullseye, 3-buster, 3.2-buster, 3.1-buster, 3.0-buster ([full list](https://mcr.microsoft.com/v2/devcontainers/ruby/tags/list)) |
```
Comment (MrNice78) [inline src/ruby/README.md:12]: [README.md](https://github.com/devcontainers/images/files/15172554/README.md)

File: src/ruby/README.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ You can directly reference pre-built versions of `Dockerfile` by using the `imag
 - `mcr.microsoft.com/devcontainers/ruby:3.3` (or `3.3-bookworm`, `3.3-bullseye` to pin to an OS version)
 - `mcr.microsoft.com/devcontainers/ruby:3.2` (or `3.2-bookworm`, `3.2-bullseye`, `3.2-buster` to pin to an OS version)
 - `mcr.microsoft.com/devcontainers/ruby:3.1` (or `3.1-bookworm`, `3.1-bullseye`, `3.1-buster` to pin to an OS version)
-- `mcr.microsoft.com/devcontainers/ruby:3.0` (or `3.0-bullseye`, `3.0-buster` to pin to an OS version)
 
 Refer to [this guide](https://containers.dev/guide/dockerfile) for more details.
```
Comment (MrNice78) [inline src/ruby/README.md:30]: [README.md](https://github.com/devcontainers/images/files/15172555/README.md)

File: src/ruby/README.md [hunk 3]
```diff
@@ -26,7 +26,6 @@ You can directly reference pre-built versions of `Dockerfile` by using the `imag
 - `mcr.microsoft.com/devcontainers/ruby:3.3` (or `3.3-bookworm`, `3.3-bullseye` to pin to an OS version)
 - `mcr.microsoft.com/devcontainers/ruby:3.2` (or `3.2-bookworm`, `3.2-bullseye`, `3.2-buster` to pin to an OS version)
 - `mcr.microsoft.com/devcontainers/ruby:3.1` (or `3.1-bookworm`, `3.1-bullseye`, `3.1-buster` to pin to an OS version)
-- `mcr.microsoft.com/devcontainers/ruby:3.0` (or `3.0-bullseye`, `3.0-buster` to pin to an OS version)
```
Comment (MrNice78) [inline src/ruby/README.md:29]: [README.md](https://github.com/devcontainers/images/files/15172558/README.md)

**Pull request merged**